integration/fuse/src/test/java/alluxio/fuse/auth/AbstractAuthPolicyTest.java

```diff
@@ -81,6 +81,11 @@ public void setUserGroup() throws Exception {
     URIStatus status = mFileSystem.getStatus(uri);
     Assert.assertEquals(userName, status.getOwner());
     Assert.assertEquals(groupName, status.getGroup());
+
+    // do not call "setAttribute" if the file already has correct owner and group
```

Comment:
Do you intend to verify the `mFileSystem.setAttribute()` is not called?

Reply:
Yes. I just adjusted the comment, please help review once more. Thank you.

Reply:
It's not easy to do mock/spy here, so I do it in this way.